code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -997,13 +997,13 @@
 
 //ACID
 
-
 /datum/reagent/toxin/acid
 	name = "Sulfuric Acid"
 	description = "A strong mineral acid with the molecular formula H2SO4."
 	color = "#00FF32"
-	toxpwr = 1
-	var/acidpwr = 10 //the amount of protection removed from the armour
+	toxpwr = 0 // Acid causes burns and brute damage, not toxin
+	penetrates_skin = NONE // Acid needs to be directly injected/ingested to metabolize (vapor/foam effects are deadly enough)
+	var/acidpwr = 10 //the amount of protection removed from the armour (also used to calculate damage if metabolized)
```

Comment:
Remove protection? What does this mean?

Reply:
Not sure, I think it originally was used to degrade armor protection values.  I used it to scale acid damage in my code.